frontend/src/slices/auth/selectors.ts

```diff
@@ -0,0 +1,11 @@
+import { type User } from 'shared/build/index.js';
+
+import { type DataStatus } from '~/libs/enums/data-status.enum.js';
+import { type RootState, type ValueOf } from '~/libs/types/types.js';
+
+const selectIsLoading = (state: RootState): ValueOf<typeof DataStatus> =>
+  state.auth.dataStatus;
```

Comment:
```suggestion
const selectIsLoading = (state: RootState): ValueOf<typeof DataStatus> => {
 return  state.auth.dataStatus;
 }
```
to avoid such line breaks